metriq_gym/benchmarks/__init__.py

```diff
@@ -1,23 +1,27 @@
 from metriq_gym.benchmarks.benchmark import Benchmark, BenchmarkData
```

Comment:
Issue for a later day, but since you alphabetize the handlers below, it would be stylistically nice/coherent to do this same for the imports. We could do this using `isort` and could just hook that up to the pre-commit hook. Not needed to address here, but if you agree, I can open a separate issue for that as a low priority.

Reply:
I mostly let my editor add and remove import automatically, so I don't care much about how they are sorted.